_rules/text-contrast-afw4f7.md

```diff
@@ -296,6 +296,16 @@ This text in a [semantic button][semantic role] has a contrast ratio of 3.85:1.
 <div role="button" style="color: #777; background: #EEE;">My button!</div>
 ```
 
+#### Failed Example 11
+
+This black text has a contrast ratio between 4.11:1 and 4.17:1 on gray background with white text shadow on it.
+
+```html
+<p style="color: #000; background: #676767; text-shadow: white 0 0 40px">
+	Some text in a human language
+</p>
```

Comment:
```suggestion
The grey text has a contrast between 2.7:1 and 2.9:1 against the grey text shadow.

```html
<p style="color: #666; text-shadow: #aaa 2px 2px 4px, #aaa -2px 2px 4px, #aaa 2px -2px 4px, #aaa -2px -2px 4px;">
    Some text in a human language
  </p>
```

Reply:
Adding the background CSS property as well, to prevent an element from having only a specified text color without a background